flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/JobMasterTest.java

```diff
@@ -1362,7 +1363,8 @@ void testRequestPartitionState() throws Exception {
             // finish the producer task
             jobMasterGateway
                     .updateTaskExecutionState(
-                            new TaskExecutionState(executionAttemptId, ExecutionState.FINISHED))
+                            SchedulerTestingUtils.createFinishedTaskExecutionState(
+                                    executionAttemptId))
```

Comment:
Because the FLINK-29664 required IOMetrics not null when mark task finished. And the older constructor will create a finished TaskExecutionState with null IOMetrics.